samples/net/wpan_serial/README.rst

```diff
@@ -0,0 +1,54 @@
+.. _wpan_serial-sample:
+
+802.15.4 "serial-radio" sample
+##############################
+
+Overview
+********
+
+The wpan_serial sample shows how to use hardware with 802.15.4 radio and USB
+controller as a "serial-radio" device for Contiki-based border routers.
+
+Requirements
+************
+
+The sample assumes that 802.15.4 radio and USB controller are supported on
+a board.
+
+Building and Running
+********************
+
+#. Before building and running this sample, be sure your Linux system's
```

Comment:
The continuation lines of a bullet list must be indented:
```
#. Before building and running this sample, be sure your Linux system's
   ModemManager is disabled, otherwise, it can interfere with serial
   port communication:
```

Reply:
fixed